core/commands/add.go

```diff
@@ -29,6 +31,7 @@ const progressReaderIncrement = 1024 * 256
 const (
 	progressOptionName = "progress"
 	wrapOptionName     = "wrap-with-directory"
+	hiddenOptionName   = "hidden"
```

Comment:
@wking 

> I'm not sure how they handle Windows' hidden attribute

Just tested this locally and `git` respects the hidden attribute. It also creates the `.git` folder as a hidden folder.